### ISSUE TITLE
Lack of screen reader feedback after pressing "Submit results" button

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-at-app#1349]
After completing the results for a single test within a plan, the natural progression is to submit the results and then move onto the next test.  To do this, testers must:
* Press the "Submit results" button;
* wait for the "Continue" button to become available; and
* press "Continue".

(I presume they could also press the "Next test" button instead of "Continue", as long as they'd submitted their results first.)

The problem is that for screen reader users, there is no feedback that the results have been submitted and the "Continue" button has become available.  Keyboard focus is lost after "Submit results" is pressed, and there is no option but to wait a bit and then go manually looking for the button.  Performing these steps many times throughout a test run is tedious.

#### Suggested Solution
When the "Continue" button becomes available, place focus on it automatically.

**Comment by ccanash** [comment]
Hi @jscholes  confirming we have triaged this. Thanks for proposing it !

**Comment by ChrisC** [comment]
@jscholes Yes, we agree the navigation here is particularly confusing. After a tester hits the "Submit results" button, the page actually reloads with new button options (including the "Continue" button), which also explains why the focus is unexpectedly lost for screen reader users. 

We would like to change this behavior and have thought of some options, none of which are necessarily mutually exclusive:

1. **Move the Continue button closer to the top of the test page**
There is unhelpful redundancy in the current placement of the "Continue" button, right next to a "Next test" button that does the same thing. There's an alert area at the top of the page (just under the top-level heading of the page) that lets testers know if they're viewing tests as another tester, or if there's conflicts for a given test. We think the "Continue" button would be more useful for quick navigation if it were moved up to this region. 

2. **Automatically advance the tester to the next test after hitting "Submit results"**
If we should not do this, and we need the tester to stay on the same test after submitting results (for example, to review the submitted results), then see suggestion #3 below. 

3. **Add an additional alert for tests that have already been submitted that reminds testers to review the results.**
This additional alert would make it more explicit to a tester that they have navigated to a test that has already been submitted. Additionally, this new alert would house a "Continue" button to quickly advance to the next test that would be easier to navigate to. 

I would recommend that we implement all three of these options to improve the test result submission experience. Let us know if you see any issues with these suggestions!

cc @mcking65
